shell/components/nav/TopLevelMenu.vue

```diff
@@ -536,6 +536,7 @@ export default {
           >
             <nuxt-link
               :to="{ name: 'about' }"
+              class="version"
```

Comment:
this looks like a dupe from above (sorry, missed from previous review)

Reply:
oh good catch! thanks for calling that out